synapseclient/core/utils.py

```diff
@@ -601,7 +642,8 @@ def itersubclasses(cls, _seen=None):
 
```

Comment:
This is a comment for the `itersubclasses` function.

If you add an indentation it'll add everything to a code block. It's rendering really oddly otherwise:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/992b80fa-4890-4be5-847d-99075ad0ae13)

Updated docstring:
```
    """
    http://code.activestate.com/recipes/576949/ (r3)

    itersubclasses(cls)

    Generator over all subclasses of a given class, in depth first order.

        >>> list(itersubclasses(int)) == [bool]
        True
        >>> class A(object): pass
        >>> class B(A): pass
        >>> class C(A): pass
        >>> class D(B,C): pass
        >>> class E(D): pass
        >>>
        >>> for cls in itersubclasses(A):
        ...     print(cls.__name__)
        B
        D
        E
        C
        >>> # get ALL (new-style) classes currently defined
        >>> [cls.__name__ for cls in itersubclasses(object)] #doctest: +ELLIPSIS
        ['type', ...'tuple', ...]
    """
```

Reply:
updated in latest commit